src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/UsePreferredTerms.cs

```diff
@@ -184,8 +184,7 @@ public abstract class UsePreferredTermsAnalyzer : DiagnosticAnalyzer
 
         public override void Initialize(AnalysisContext analysisContext)
         {
-            // TODO: Enable concurrent execution of analyzer actions.
-            //analysisContext.EnableConcurrentExecution();
+            analysisContext.EnableConcurrentExecution();
```

Comment:
Has no effect - analyzer reports no supported diagnostics, and this should never be called.

Reply:
That seems fine. If it ever is implemented, thread safe wouldn't be optional.